sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -50,7 +53,16 @@ class SampleApplication : Application() {
     )
 
     private val okHttpClient = OkHttpClient.Builder()
-        .addInterceptor(RumInterceptor())
+        .addInterceptor(
+            RumInterceptor(
+                rumResourceAttributesProvider = object : RumResourceAttributesProvider {
+                    override fun onProvideAttributes(
+                        request: Request,
+                        response: Response?,
+                        throwable: Throwable?
+                    ): Map<String, String> = mapOf("foo" to "bar")
```

Comment:
hmmm...I'd rather test it if it works but not add those in the sample app. A good place where you can add this and you could do this in a next task is in the `integration tests` project and try to assert for them in the mock server results.